src/boss/cloudvm/worker_config_template.go

```diff
@@ -32,6 +40,30 @@ func LoadWorkerConfigTemplate(templatePath string, workerPath string) error {
 		return fmt.Errorf("failed to load template.json: %v", err)
 	}
 
+	// Patch fields ONLY if they're empty
```

Comment:
If they're empty, it makes sense to plug in default values.  But can we pull that from the worker defaults instead of hardcoding that here?  E.g., if we want to change the default import cache to be a different zygotes file, I don't want to have to change the code here and in this file: https://github.com/open-lambda/open-lambda/blob/main/src/common/config.go#L143